tests/integration/testdata/buildcmd/terraform/zip_based_lambda_functions_local_backend/main.tf

```diff
@@ -365,7 +374,81 @@ resource "aws_lambda_layer_version" "layer4" {
       null_resource.build_layer4_version, aws_s3_object.layer4_code
   ]
 }
-## */ layer3
+## */ layer4
+
+## /* layer5
+resource "null_resource" "build_layer5_version" {
+    triggers = {
+        build_number = "${timestamp()}"
+    }
+
+    provisioner "local-exec" {
+        command = "./py_build.sh \"${local.layer5_src_path}\" \"${local.building_path}\" \"${local.layer5_artifact_file_name}\" Layer"
+    }
+}
+
+resource "null_resource" "sam_metadata_aws_lambda_layer_version_layer5" {
+    triggers = {
+        resource_type = "LAMBDA_LAYER"
+        original_source_code = local.layer5_src_path
+        built_output_path = "${local.building_path}/${local.layer5_artifact_file_name}"
+    }
+    depends_on = [
+        null_resource.build_layer5_version
+    ]
+}
+
+resource "aws_s3_object" "layer5_code" {
+  bucket = "existing_s3_bucket_name"
+  key    = "layer5_code"
+  source = "${local.building_path}/${local.layer5_artifact_file_name}"
+  depends_on = [
+    null_resource.build_layer5_version
+  ]
+}
+
+resource "aws_lambda_layer_version" "layer5" {
+  s3_bucket = "existing_s3_bucket_name"
+  s3_key = "layer5_code"
+  layer_name = "lambda_layer5"
+  compatible_runtimes = ["python3.8"]
+  depends_on = [
+      null_resource.build_layer5_version, aws_s3_object.layer5_code
+  ]
+}
+## */ layer5
+
+## /* layer6
+resource "null_resource" "build_layer6_version" {
+    triggers = {
+        build_number = "${timestamp()}"
+    }
+
+    provisioner "local-exec" {
+        command = "./py_build.sh \"${local.layer6_src_path}\" \"${local.building_path}\" \"${local.layer6_artifact_file_name}\" Layer"
```

Comment:
Since we reference `./py_build.sh` ,do we have a similar test for windows?

Reply:
yes